app/src/main/java/com/framgia/englishconversation/data/model/TimelineModel.java

```diff
@@ -227,6 +238,20 @@ public int getViewType() {
         return getMedias() != null ? getMedias().size() : 0;
     }
 
+    @Override
```

Comment:
WARNING: Definition of 'equals()' without corresponding definition of 'hashCode()'.